drivers/media/platform/virtio/virtio-camera.c

```diff
@@ -157,6 +157,7 @@ static void virtio_camera_control_ack(struct virtqueue *vq)
 			vbuf->sequence = req->resp.u.buffer.sequence;
 			vbuf->vb2_buf.timestamp = req->resp.u.buffer.timestamp;
 			vbuf->planes[0].bytesused = req->resp.u.format.size.sizeimage;
+			req->vb->planes[0].bytesused = req->resp.u.format.size.sizeimage;
```

Comment:
I think line159 is enough to provide the bytesused for userspace. 
Do you verify the android image released form this week or last week. It should work. 


buffer info in this struct can be seen from userspace.
struct vb2_v4l2_buffer {
	struct vb2_buffer	vb2_buf;

	__u32			flags;
	__u32			field;
	struct v4l2_timecode	timecode;
	__u32			sequence;
	__s32			request_fd;
	bool			is_held;
	struct vb2_plane	planes[VB2_MAX_PLANES];
	__u32                   reserved;

	ANDROID_KABI_RESERVE(1);
};